app/experimenter/nimbus-ui/src/components/PageEditAudience/index.test.tsx

```diff
@@ -11,20 +11,33 @@ import PageEditAudience from "src/components/PageEditAudience";
 import FormAudience from "src/components/PageEditAudience/FormAudience";
 import { UPDATE_EXPERIMENT_MUTATION } from "src/gql/experiments";
 import { CHANGELOG_MESSAGES, SUBMIT_ERROR } from "src/lib/constants";
-import { mockExperimentQuery } from "src/lib/mocks";
+import { mockExperimentQuery, mockLiveRolloutQuery } from "src/lib/mocks";
 import { RouterSlugProvider } from "src/lib/test-utils";
 import {
   ExperimentInput,
   NimbusExperimentChannelEnum,
   NimbusExperimentFirefoxVersionEnum,
+  NimbusExperimentPublishStatusEnum,
+  NimbusExperimentStatusEnum,
 } from "src/types/globalTypes";
 import { updateExperiment_updateExperiment } from "src/types/updateExperiment";
 
 const { mock, experiment } = mockExperimentQuery("demo-slug");
+const { mockRollout, rollout } = mockLiveRolloutQuery("demo-slug-1");
 
 let mockSubmitData: Partial<ExperimentInput>;
 let mutationMock: ReturnType<typeof mockUpdateExperimentAudienceMutation>;
 
+let mockRolloutSubmitData: Partial<ExperimentInput>;
+let rolloutMutationMock: ReturnType<
+  typeof mockUpdateExperimentAudienceMutation
+>;
+
+let mockDirtyRolloutSubmitData: Partial<ExperimentInput>;
+let dirtyRolloutMutationMock: ReturnType<
+  typeof mockUpdateExperimentAudienceMutation
+>;
```

Comment:
Wow.  Wow.   Bless your heart.  All I've been doing for weeks is trying to hunt down all the places where `Record<string, any>` is written in stead of `ExperimentInput`.  Thank you for using the actual type 🙏 😭